src/App.tsx

```diff
@@ -10,31 +11,67 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
+  const [selectedTabId, onTabSelected] = useState('tab-1');
+  let selectedTitle = 'Tab 1';
+
+  const preventDef = (event: MouseEvent) => {
+    event.preventDefault();
+  };
+
+  tabs.map((tab) => {
+    if (selectedTabId === tab.id) {
+      selectedTitle = tab.title;
+    }
+
+    return 0;
+  });
+
+  useEffect(() => {
+    window.addEventListener('click', preventDef);
+  }, []);
```

Comment:
Don't add preventDefault action to all clicks on the page. Seems that you don't need any preventDefault actions in this task at all
```suggestion
```